resin/models/supervisor.py

```diff
@@ -0,0 +1,290 @@
+                'method': 'GET'
+            }
+
+            return self.base_request.request(
+                'supervisor/ping',
+                'POST',
+                data=data,
+                endpoint=self.settings.get('api_endpoint'),
+                login=True
+            )
+        else:
+            return self.base_request.request(
+                'ping',
+                'GET',
+                endpoint=self.RESIN_SUPERVISOR_ADDRESS,
+                api_key=self.RESIN_SUPERVISOR_API_KEY
+            )
+
+    def blink(self, device_id=None, app_id=None):
+        """
+        Start a blink pattern on a LED for 15 seconds. This is the same with `resin.models.device.identify()`.
+        No need to set device_id and app_id if command is sent to the API on device.
+
+        Args:
+            device_id (Optional[str]): device id.
+            app_id (Optional[str]): application id.
+
+        Raises:
+            InvalidOption: if the endpoint is Resin API proxy endpoint and device_id or app_id is not specified.
+
+        """
+
+        self._do_command(
+            '{0}/blink'.format(self.SUPERVISOR_API_VERSION),
+            device_id=device_id,
+            app_id=app_id,
+            method='POST'
+        )
+
+    def update(self, device_id=None, app_id=None, force=False):
+        """
+        Triggers an update check on the supervisor. Optionally, forces an update when updates are locked.
+        No need to set device_id and app_id if command is sent to the API on device.
+
+        Args:
+            device_id (Optional[str]): device id.
+            app_id (Optional[str]): application id.
+            force (Optional[bool]): If force is True, the update lock will be overridden.
+
+        Raises:
+            InvalidOption: if the endpoint is Resin API proxy endpoint and device_id or app_id is not specified.
+
+        """
+
+        if force:
+            data = {
+                'force': force
+            }
+        else:
+            data = {}
+
+        self._do_command(
+            '{0}/update'.format(self.SUPERVISOR_API_VERSION),
+            req_data=data,
+            device_id=device_id,
+            app_id=app_id,
+            method='POST'
+        )
+
+    def reboot(self, device_id=None, app_id=None):
+        """
+        Reboot the device.
+        No need to set device_id and app_id if command is sent to the API on device.
+
+        Args:
+            device_id (Optional[str]): device id.
+            app_id (Optional[str]): application id.
+
+        Returns:
+            dict: when successful, this dictionary is returned `{ 'Data': 'OK', 'Error': '' }`.
+
+        Raises:
+            InvalidOption: if the endpoint is Resin API proxy endpoint and device_id or app_id is not specified.
+
+        """
+
+        self._do_command(
+            '{0}/reboot'.format(self.SUPERVISOR_API_VERSION),
+            device_id=device_id,
+            app_id=app_id,
+            method='POST'
+        )
```

Comment:
You say it returns a dict in the method's docstring, but there's no ``return something`` :p